hubblestack/extmods/utils/customUserConf.py

```diff
@@ -0,0 +1,59 @@
+import json
+import yaml
+import logging
+base_path_linux = '/etc/hubble/hubble.d/'
+base_path_windows = 'C:\\Program Files (x86)\\Hubble\\etc\\hubble\\hubble.d\\'
+log = logging.getLogger(__name__)
+default_config_name = 'user.conf'
+
+def parse_configure_option(configure_options):
+    try:
+        keys = configure_options.split(" ")
+        configure_options_dict = {}
+        for str in keys:
+            key = str.split("=")[0]
+            value = str.split("=")[1]
+            configure_options_dict[key] = value
+    except:
+        raise Exception('some error occured, parameters format incorrect')
+    return configure_options_dict
+
+
+def create_user_conf(splunk_conf, configure_options, is_windows):
+    '''
+    Function to create a user conf based on the parameters provided in the
+    command line.
+    Usage : hubble --configure "splunk_token=customToken splunk_index=customIndex confname=mycustom.conf"
+    :param splunk_conf: splunk config as mentioned in the main hubble conf
+    :param configure_options: command line args passed to the --configure method
+    :param is_windows: boolean that tells whether the OS is windows or not
+    :return: void
+    '''
+    print('creating custom conf')
```

Comment:
We need to remove these `print` statements. For one, they are in python3 form without doing the proper `from __future__ import print_function`. Additionally, we want to do all of our emitting via logging instead. @jettero set up [an early logging handler](https://github.com/hubblestack/hubble/blob/fb8f89aaed3ad8797318a0f97c0f0a230a27f00b/hubblestack/log.py#L82-L87) so you can log even at this early stage.